blog/releases/announcing-knative-v0-22-release.md

```diff
@@ -0,0 +1,195 @@
+
+#### 🐞 Bug Fixes
+
+- Adds validation that a default max-scale is set if a max-scale-limit is specified in the autoscaler configmap (since otherwise the default max-scale, i.e. 0 = no max, would fail validation as it is above the max-scale-limit). [#10921](https://github.com/knative/serving/pull/10921)
+- Bumped the resource request and limits of the autoscaler to 100m/100Mi, 1000m/1000Mi respectively. [#10865](https://github.com/knative/serving/pull/10865)
+- Fixed a regression where the pod bringup time might have a latency of 10s or more even though the container should be up quickly. [#10992](https://github.com/knative/serving/pull/10992)
+- Reduced the necessary memory allocations in the activator significantly, especially with disabled tracing. [#11016](https://github.com/knative/serving/pull/11016), [#11013](https://github.com/knative/serving/pull/11013), [#11009](https://github.com/knative/serving/pull/11009), [#11008](https://github.com/knative/serving/pull/11008)
+- Fix the incorrect Gateway name format for DomainMapping auto TLS feature for net-istio implmenetation. [net-istio#532](https://github.com/knative-sandbox/net-istio/pull/532)
+
+
+### Eventing v0.22
+
+#### 🚨 API Changes
+
+- v1alpha1 Channel duck types are no longer supported. [#5005](https://github.com/knative/eventing/pull/5005)
+- Allow Trigger users to set another namespace's Subscriber, And when Subscriber's namespace not been set, it will be set to Trigger's namespace [#4995](https://github.com/knative/eventing/pull/4995)
+
+#### 🐞 Bug Fixes
+
+- Fix PingSource adapter not always releasing the leader election lease when terminating [#5162](https://github.com/knative/eventing/pull/5162)
+- Fix PingSource sending duplicate events when the leading adapter fails to renew its lease [#4908](https://github.com/knative/eventing/pull/4908)
+- Fix bug preventing the namespace-scoped in-memory channel to become ready. [#4906](https://github.com/knative/eventing/pull/)
+- Fix excessive number of Replicas for eventing-webhook [#5112](https://github.com/knative/eventing/pull/5112)
+- Webhook timeout for API server calls has been increased to 10 seconds. [#5175](https://github.com/knative/eventing/pull/5175)
+
+#### 🧹 Clean up
+
+- Do not set the finalizer for pingSource for pingSource controller, just deal with it in reconcilekind [#5002](https://github.com/knative/eventing/pull/5002)
+
+### Eventing Extensions
+
+
+#### Eventing Kafka Broker v0.22
+
+#### 🚨 Breaking or Notable
+
+- Kubernetes's minimum version is 1.18. [#779](https://github.com/knative-sandbox/eventing-kafka-broker/pull/779)
+
+#### 💫 New Features & Changes
+
+- Support ordered delivery.
+  - You can choose between ordered and unordered delivery through the label kafka.eventing.knative.dev/delivery.order in your triggers.
+  - Check out the docs for more details at https://knative.dev/docs/eventing/broker/kafka-broker . [#589](https://github.com/knative-sandbox/eventing-kafka-broker/pull/)
```

Comment:
Again, I feel like this was mentioned earlier but in less detail. We need a better overall structure for this doc, but we can follow that up in a future release. Won't block the PR over it.